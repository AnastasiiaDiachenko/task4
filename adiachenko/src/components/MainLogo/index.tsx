import React from "react";
import styled from 'styled-components';
import { variables } from '../../helpers/styleVariables';

const Logo = styled.span`
  color: #${variables.colors.LOGO};
  font-size: ${variables.fontSize.LOGO};
  z-index: 3;
`;

const LogoBold = styled.span`
  font-weight: bold;
`;

const MainLogo = () => (
  <Logo>
    <LogoBold>netflix</LogoBold>roulette
  </Logo>
);

export default MainLogo;
