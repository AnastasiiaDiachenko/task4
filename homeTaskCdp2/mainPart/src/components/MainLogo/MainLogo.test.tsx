import React from 'react';
// @ts-ignore
import renderer from 'react-test-renderer';
import MainLogo from './index';

describe('<MainLogo />', () => {
  it('should renders correctly when user is on logged in page', () => {
    const tree = renderer
      .create(<MainLogo />)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
