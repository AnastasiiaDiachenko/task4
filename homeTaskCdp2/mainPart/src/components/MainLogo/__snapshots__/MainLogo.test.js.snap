// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Header /> should renders correctly when user is on logged in page 1`] = `
<span
  className="sc-bdfBwQ dvqrJZ"
>
  <span
    className="sc-gsTCUz bCQwq"
  >
    netflix
  </span>
  roulette
</span>
`;

exports[`<MainLogo /> should renders correctly when user is on logged in page 1`] = `
<span
  className="sc-bdfBwQ dvqrJZ"
>
  <span
    className="sc-gsTCUz bCQwq"
  >
    netflix
  </span>
  roulette
</span>
`;
